ci/redmine_bugzilla.py

```diff
@@ -250,7 +250,10 @@ def main():
                 if external_bug['type']['description'] == 'Pulp Redmine':
                     add_cc_list_to_bugzilla_bug(bug)
                     issue_id = external_bug['ext_bz_bug_id']
-                    issue = redmine.issue.get(issue_id)
+                    try:
+                        issue = redmine.issue.get(issue_id)
+                    except exceptions.ResourceNotFoundError:
+                        print "Issue ID {} linked as external on BZ {} does not exist".format(issue_id, bug.id)
```

Comment:
I think this block should re-raise. Otherwise you'll get an exception a couple lines later about reference before assignment.

Reply:
Resolved